docs/installation.rst

```diff
@@ -0,0 +1,188 @@
+Installation
+===============
+
+Local Kubernetes installation
+-----------------------------
+
+Requirements
+````````````
+In order to create a local REANA cluster with Kubernetes as a backend
+the following requirements must be met:
+
+- `VirtualBox`
+- `VT-x/AMD-v` virtualization must be enabled in BIOS
+
+Quickstart
+``````````
+
+We start by creating a fresh new Python virtual environment that will handle our `REANA` cluster instantiation.
+
+.. code-block:: console
+
+   $ mkvirtualenv reana
+
+We update ``pip`` and ``setuptools``
+
+.. code-block:: console
+
+   $ pip install --upgrade pip setuptools
+
+Next we install ``reana`` package
+
+.. code-block:: console
+
+   $ git clone https://github.com/reanahub/reana
+   $ cd reana
+   $ pip install -e .[all]
+
+Now, for instantiating a local REANA `minikube` based cluster we should run
+
+.. code-block:: console
+
+   $ reana install-minikube
+
+which will do the following:
```

Comment:
It would be good to decouple the steps that install kubernetes/minikube itself, which many people would prefer to do outside of REANA using their packaging systems, and install the rest of the REANA component ecosystem.

The getting started overview for people would then look like:

```
$ pip install reana
$ reana install-backend minikube # or skip this step if you have it already
$ reana configure-backend minikube # or other clouds later... just an example
$ reana configure-container docker # or singularity... just an example
$ reana pull-components 
$ reana build-manifests
$ reana initialise-backend
$ reana run-hello-world
...
```

(hypothetical example, let's muse IRL)